compiler/rustc_baked_icu_data/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+//! Baked ICU data for eager translation support.
+//!
+#![allow(elided_lifetimes_in_paths)]
+
+/* generated with:
```

Comment:
nit: maybe this comment could be part of the documentation comment for the crate, rather than just in the `lib.rs` file?

It might be nice to include a little bit more detail in that comment about why this crate exists, what the data in it is, when it needs regenerated, etc.

Reply:
"This crate contains pre-baked ICU4X data, generated by icu4x-datagen. The tool fetches locale data from CLDR and transforms it into `const` code in statics that ICU4X can load, via `databake`. lib.rs in this crate is manually written, but all other code is generated.

This crate can be regenerated when there's a new CLDR version, though that is unlikely to result in changes in most cases (currently this only covers list formatting data, which is rather stable). It may need to be regenerated when updating ICU4X versions, especially across major versions, in case it fails to compile after an update.

It must be regenerated when adding new locales to Rust, or if Rust's usage of ICU4X grows to need more kinds of data
"

Reply:
Thanks!

Reply:
Updated the crate documentation.